docs/components/modeler/web-modeler/advanced-modeling/form-linking.md

```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
 This conversion will only be applied to the XML deployed to the cluster; the diagram in Web Modeler will not be changed.
 :::
 
+#### Known issues with linked forms
+
+##### Wrong form used for user task - [zeebe/#16311](https://github.com/camunda/zeebe/issues/16311)
```

Comment:
H4 headers and beyond make the text quite small, so launching an issue in my backlog to address this across our docs.